hazelcast/src/test/java/com/hazelcast/topic/TopicTest.java

```diff
@@ -83,19 +82,21 @@ public void run() {
 
     @Test
     public void testTopicPublishingMember() {
-        final Config config = new Config();
+        final int nodeCount = 3;
+
+        Config config = new Config();
         config.getTopicConfig("default").setGlobalOrderingEnabled(true);
-        final int k = 3;
-        TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(k);
-        final HazelcastInstance[] instances = factory.newInstances(config);
 
-        final CountDownLatch mainLatch = new CountDownLatch(k);
-        final AtomicInteger count = new AtomicInteger(0);
+        TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(nodeCount);
+        HazelcastInstance[] instances = factory.newInstances(config);
+
+        final CountDownLatch mainLatch = new CountDownLatch(nodeCount);
         final AtomicInteger count1 = new AtomicInteger(0);
         final AtomicInteger count2 = new AtomicInteger(0);
+        final AtomicInteger count3 = new AtomicInteger(0);
         final String name = "testTopicPublishingMember";
```

Comment:
Could it be that
```
config.getTopicConfig("default").setGlobalOrderingEnabled(true);
```
has no effect, because we use another topicName later on?
```
final String name = "testTopicPublishingMember";
```
I think this maybe a copy&paste error from another test.